### PR TITLE
Feat/carousel

### LINKED ISSUE #83: ### Carousel

### PR DESCRIPTION
**Pull request by Liiked**
- [ ] Component Migration

Fix #83

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
https://deploy-preview-151--element-plus.netlify.app/?path=/story/carousel--basic-usage
Doc site is full of errors, fix them first

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "sass": "^1.26.10",
     "sass-loader": "^9.0.3",
     "style-loader": "^1.2.1",
-    "throttle-debounce": "1.0.1",
+    "throttle-debounce": "^2.2.1",
```
Comment (hanx316) [inline package.json:52]: Since we are already using lodash, is there any reason to import this package?

Comment (zazzaz) [reply package.json:52]: no,  this is just the previous website's dependency, we will remove it later.

Comment (hanx316) [reply package.json:52]: OK，I suggest that we can make a refactor todo list.

**Comment by Liiked** [comment]
> https://deploy-preview-151--element-plus.netlify.app/?path=/story/carousel--basic-usage
> image
> 
> Doc site is full of errors, fix them first

errors has been fixed, review the code at your convinence😉

**Comment by hanx316** [comment]
Hi, please resolve the conflicts or you may update your feat branch with the latest dev.

**Review by jw-foss — APPROVED** [review]

**Pull request merged**